### DISCUSSION TITLE
libicu71 not available

### DISCUSSION THREAD

**Discussion by fethomm** [cboxdoerfer/fsearch#403] [Q&A]
Hi, 
just tried  to install fsearch on Ubuntu Jammy. Install fails because libicu71 is not available.

**Comment by cboxdoerfer** [comment]
Hi, that's weird. Do you maybe accidentally use the Ubuntu Kinetic PPA somehow (the FSearch package from there was built against and requires ICU 71)?

**Reply by fethomm** [reply]
I used what is offered on https://launchpad.net/~christian-boxdoerfer/+archive/ubuntu/fsearch-stable
sources.list.d says: 
stable.jammy.list
also tried:
daily.jammy.list

**Reply by cboxdoerfer** [reply]
Can you post the output of `apt list fsearch -a` ?

**Comment by fethomm** [comment]
ft  tux  ~  apt list fsearch -a
Auflistung… Fertig
fsearch/jammy 20220829+r2010.df06e31~ubuntu22.04.1 amd64
fsearch/unbekannt 0.2.2-1+3.4 amd64
fsearch/jammy 0.2.2-0~202208231101~ubuntu22.04.1 amd64

I have built the app myself meantime, That worked like a charm. This Ubuntu-based laptop was the last device without your fabulous search tool :)

**Reply by cboxdoerfer** [reply]
> I have built the app myself meantime, That worked like a charm.

:+1: 

> fsearch/unbekannt 0.2.2-1+3.4 amd64

That's probably the reason why it didn't work — apt tried to install this package instead of the ones from the PPAs. `apt-cache policy fsearch` should tell you from which repository this package comes, and if you don't need that repository you can simply remove it.

**Comment by fethomm** [comment]
You nailed it. I had eroniously assumed for a start, that this machine was debian and went to OpenBuild. So this repo was a remnant of that. My bad.

**Reply by cboxdoerfer** [reply]
No problem.
